src/frontend/src/pages/WorkPackageDetailPage/ActivateWorkPackageModalContainer/ActivateWorkPackageModal.tsx

```diff
@@ -91,97 +94,86 @@ const ActivateWorkPackageModal: React.FC<ActivateWorkPackageModalProps> = ({
   };
 
   return (
-    <form id={'activate-work-package-form'} onSubmit={handleSubmit(onSubmitWrapper)}>
-      <Dialog open={modalShow} onClose={onHide}>
-        <DialogTitle>{`Activate #${wbsPipe(wbsNum)}`}</DialogTitle>
-        <DialogContent>
-          <Grid container spacing={2}>
-            <Grid item xs={6}>
-              <NERAutocomplete
-                id="project-lead-autocomplete"
-                onChange={(_event, value) => setLeadId(value?.id)}
-                options={allUsers.map((p) => ({
-                  label: fullNamePipe(p),
-                  id: p.userId.toString()
-                }))}
-                size="small"
-                placeholder="Project Lead"
-                listboxProps={{ style: { maxHeight: '199px' } }}
-              />
-            </Grid>
-            <Grid item xs={6}>
-              <NERAutocomplete
-                id="project-manager-autocomplete"
-                onChange={(_event, value) => setManagerId(value?.id)}
-                options={allUsers.map((p) => ({
-                  label: fullNamePipe(p),
-                  id: p.userId.toString()
-                }))}
-                size="small"
-                placeholder="Project Manager"
-                listboxProps={{ style: { maxHeight: '199px' } }}
+    <NERFormModal
+      open={modalShow}
+      onHide={onHide}
+      title={`Activate #${wbsPipe(wbsNum)}`}
+      reset={() => reset(defaultValues)}
+      submitText="Submit"
+      handleUseFormSubmit={handleSubmit}
+      onFormSubmit={onSubmitWrapper}
+      formId="activate-work-package-form"
+      showCloseButton
+    >
+      <Grid container spacing={2}>
+        <Grid item xs={6}>
+          <NERAutocomplete
```

Comment:
Use the `errorMessage` prop so that the errors appear on the modal

Reply:
I just did it, but since the button is greyed out, the user isn't able to submit anyways without filling out the NERAutocompletes anyways

Reply:
That's fine, we want the user to also know why they can't submit